pkg/smartcontract/parameter.go

```diff
@@ -54,7 +54,9 @@ func (p Parameter) MarshalJSON() ([]byte, error) {
 	)
 	if p.Value == nil {
 		if _, ok := validParamTypes[p.Type]; ok && p.Type != UnknownType {
-			return json.Marshal(rawParameter{Type: p.Type})
+			return json.Marshal(rawParameter{Type: p.Type, Value: []byte("null")})
```

Comment:
What's the `p.Type` in this case?

Reply:
It’s Any, isn’t it?

Reply:
All other types have their values filled in.

Reply:
Technically you can some other type with `Value == nil`. My concern is can we get some `Integer` or `Bool` with `null` value this way. Looks like we can. How will it be processed on the other side? Probably fail. Then should we allow this behavior?